src/test/resources/clickhouse_cgds.sql

```diff
@@ -206,7 +206,8 @@ CREATE TABLE genetic_profile
     description                   String,
     show_profile_in_analysis_tab  Int32,
     generic_assay_type            Nullable(String),
-    cancer_study_id               Int32
+    cancer_study_id               Int32,
+    patient_level                 Nullable(Int32),
```

Comment:
Does this need to be nullable? In clickhouse it creates a separate Nullable column to handle this? Just want to make sure because it will increase the data foot print